pkg/agent/openflow/service.go

```diff
@@ -124,6 +124,15 @@ func newFeatureService(
 	}
 }
 
+// serviceNoEndpointFlow generates the flow to match the packets to Service without Endpoint and send them to controller.
+func (f *featureService) serviceNoEndpointFlow() binding.Flow {
+	return EndpointDNATTable.ofTable.BuildFlow(priorityLow).
```

Comment:
No special reason, both `priorityLow` and `priorityHigh` can work since the reg mark is unique.
```
       ServiceEPStateField = binding.NewRegField(4, 16, 18)
	EpToSelectRegMark   = binding.NewRegMark(ServiceEPStateField, 0b001)
	EpSelectedRegMark   = binding.NewRegMark(ServiceEPStateField, 0b010)
	EpToLearnRegMark    = binding.NewRegMark(ServiceEPStateField, 0b011)
	NoEpToSelectRegMark = binding.NewRegMark(ServiceEPStateField, 0b100)
```

Reply:
I feels "priorityHigh" is more appropriate for such case that the match condition is static and no conflict with other flow. OVS is always trying to match a packet from higher priority to lower.

Reply:
> OVS is always trying to match a packet from higher priority to lower.

From my understanding, for a packet should be rejected, if it is priorityHigh, it could be matched earlier without matching other flows, but in this way, other packets will be also matched with the flow for rejecting packet first.  Is that right?

Reply:
Why do other packets match the flow? Other packets does not contain the regmark

Reply:
shouldn't it use priorityNormal if there is no special reason we want it to be prioritized higher or lower?

Reply:
After discussed with @wenyingd offline, both `priorityNormal` and `priorityHight` are okay. Since this is related to Endpoint selection, `priorityNormal` is better like other Endpoint selection flows.